src/AccessToken.php

```diff
@@ -266,20 +340,24 @@ private function retrieveCertsFromLocation($url, array $options = [])
         ), $response->getStatusCode());
     }
 
-    /**
-     * Set required defaults for JWT.
-     */
-    private function configureJwtService()
+    private function checkAndInitializePhpsec()
     {
-        $class = class_exists('Firebase\JWT\JWT')
-            ? 'Firebase\JWT\JWT'
-            : '\JWT';
+        // @codeCoverageIgnoreStart
+        if (!class_exists(RSA::class)) {
```

Comment:
We didn't fail if phpseclib was uninstalled before. Was that incorrect/broken behavior?

Reply:
It would have thrown a PHP Fatal error "class not found" instead of an exception. I think this behavior is better, but I can revert it to uphold the almighty Backwards Compatibility!